src/Analyzers/CSharp/Tests/UseExpressionBodyForLambda/UseExpressionBodyForLambdasAnalyzerTests.cs

```diff
@@ -176,61 +177,7 @@ void Goo()
     }
 
     void TargetMethod(Func<int, string> targetParam) { }
-}", options: UseBlockBody);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExpressionBody)]
-        public async Task UseExpressionBodyFromReturnKeyword()
-        {
-            await TestMissingInRegularAndScriptAsync(
-@"using System;
-
-class C
-{
-    void Goo()
-    {
-        Func<int, string> f = x =>
-        {
-            [|return|] x.ToString();
-        };
-    }
-}", new TestParameters(options: UseExpressionBody));
```

Comment:
The deletion of the tests should be okay.

The old test harness works as "invoke lightbulb on this position".
This is conceptually different than the new testing library which asserts diagnostics locations. This also means that we know codefix is only available in the span of diagnostic. These are already tested in `UseExpressionBodyInMethod`.